runtime/buffer.cc

```diff
@@ -661,7 +661,7 @@ SLINKY_NO_STACK_PROTECTOR SLINKY_ALWAYS_INLINE inline void for_each_impl(span<co
     bases[n] = bufs[n]->base;
   }
 
-  for_each_loop_impl<BufsSize> inner_impl;
+  for_each_loop_impl<BufsSize> inner_impl = nullptr;
```

Comment:
I really wanted to leave this uninitialized, I'm trying to optimize every last cycle from this function.